README.md

```diff
@@ -87,39 +101,99 @@ The VTS is a metric designed to help stakers evaluate the performance and reliab
+- `missed`: The number of batches where at least a slot was missed.
+- `rewarded`: The number of batches rewarded.
+- `likelihood`: The probability of being selected to produce a block, calculated by `numSlots / SLOTS`.
+- `dominanceViaSlots`: Same as `likelihood`
+- `dominanceViaBalance`: The dominance ratio of the validator, calculated by `validatorBalance / totalBalance`. Might be -1 if the `balance` was not fetched when the epoch was active.
 
-### Database
+#### Current epoch
 
-As well as storing the [Validator Details](#validator-details), we also store the data produced by the fetcher in a sqlite database. This data is then used in the score calculator to compute the score. You can see the file [schema.ts](./server/database/schema.ts).
+It fetches the data from the current active epoch and stores it in the database the following variables:
 
-#### Development
+- `epoch`: The epoch number.
+- `validatorAddress`: The address of the validator.
+- `balance`: The balance of the validator.
+- `stakers`: The amount of stakers in the validator.
+
+It will also set empty values for `missed`, `rewarded`, `likelihood`, and `dominanceViaBalance` for the current epoch. This is because the epoch is still active and the data is not available yet. Those fields will be updated once the epoch ends and only if the validator was selected to produce a block, otherwise they will remain untouched.
+
+#### Types of validators
+
+| Type                          | Elected | Tracked |
+| ----------------------------- | ------- | ------- |
+| `ElectedTrackedValidator`     | ✅      | ✅      |
+| `ElectedUntrackedValidator`   | ✅      | ❌      |
+| `UnelectedTrackedValidator`   | ❌      | ✅      |
+| `UnelectedUntrackedValidator` | ❌      | ❌      |
+
+> [!NOTE]
+> Having a `UnelectedUntrackedValidator` should only happen when the validator has been selected for the first time in history.
+
+## Development
 
 Once it is cloned and installed the dependencies, you must run:
 
 ```bash
 pnpm db:generate
 pnpm dev # or pnpm dev:local to use the local database
 ```
+
+## Deployment
+
+The deployment is handled by the [`NuxtHub Action`](./.github/workflows/nuxt-hub.yml).
+
+There are 4 different environments:
+
+| Nuxt Hub Env | GitHub Env           | Dashboard URL                                                              | Trigger                       |
+| ------------ | -------------------- | -------------------------------------------------------------------------- | ----------------------------- |
+| `production` | `production-mainnet` | [Validators API Mainnet](https://validators-api-mainnet.pages.dev)         | Push to `main` branch         |
+| `production` | `production-testnet` | [Validators API Testnet](https://validators-api-testnet.pages.dev)         | Push to `main` branch         |
+| `preview`    | `preview-mainnet`    | [Validators API Mainnet Preview](https://validators-api-preview.pages.dev) | Push any commit to any branch |
```

Comment:
I guess this "previews" will be created after the PR is merged? Because they doesn't exist right now